engine/src/exec.h

```diff
@@ -2449,8 +2449,9 @@ void MCInterfaceExecReplaceInField(MCExecContext& ctxt, MCStringRef p_pattern, M
 
 void MCInterfaceExecChooseTool(MCExecContext& ctxt, MCStringRef p_input, int p_tool);
 
-void MCInterfaceExecGoCardAsMode(MCExecContext& ctxt, MCCard *p_card, int p_mode, bool p_visible, bool p_this_stack);
-void MCInterfaceExecGoCardInWindow(MCExecContext& ctxt, MCCard *p_card, MCStringRef p_window, bool p_visible, bool p_this_stack);
+enum MCGoStackVisibility { kImplicit, kExplicitVisible, kExplicitInvisible };
```

Comment:
Enums should be laid out as:
```
enum <name>
{
...
};
```
i.e. always prefer multi-line constructs, and always ensure braces are on their own line (this is generally true for new code - when merging in with older code it depends on more subjective assessment of readability of the resulting code).

Also the name should be MCInterfaceExecGoStackVisibility, and the names of the enumeration values should be prefixed by 'kMCInterfaceExecGoStackVisibility.

The reason for the above is that all these things exist in the global namespace - so brevity only potentially causes namespace conflicts later on!